torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9168,13 +9169,18 @@ def ref_pairwise_distance(input1, input2):
                DecorateInfo(unittest.skip("Skipped!"), 'TestNNCOpInfo', 'test_nnc_correctness', dtypes=(torch.half,)),
            )),
     BinaryUfuncInfo('complex',
-                    dtypes=floating_types(),
+                    dtypes=floating_types_and(torch.half),
                     supports_forward_ad=True,
                     supports_fwgrad_bwgrad=True,
                     supports_rhs_python_scalar=False,
                     skips=(
                         # RuntimeError: Expected object of scalar type Float but got scalar type Double for second argument
                         DecorateInfo(unittest.skip("Skipped!"), 'TestBinaryUfuncs', 'test_type_promotion'),
+                        # File "test/test_binary_ufuncs.py", line 334, in test_batch_vs_slicing
+                        # actual = torch.stack(actual)
+                        # RuntimeError: "cat_cuda" not implemented for 'ComplexHalf'
+                        DecorateInfo(unittest.expectedFailure, 'TestBinaryUfuncs', 'test_batch_vs_slicing',
```

Comment:
what's `test_batch_vs_slicing`?

Reply:
This is the test. It fails on CUDA because it uses `stack` which uses `cat` internally.

https://github.com/pytorch/pytorch/blob/7235ebc5e2aaec735d6ec1649c232db73b8e1698/test/test_binary_ufuncs.py#L324-L336